src/main/rx/spektrum.h

```diff
@@ -20,5 +20,5 @@
 #define SPEKTRUM_SAT_BIND_DISABLED 0
 #define SPEKTRUM_SAT_BIND_MAX 10
 
-uint8_t spektrumFrameStatus(void);
```

Comment:
Why should we do this additional scaling here? This will require RX configuration changes for everyone using Spektrum satellites and breaks compatibility for no real reason. In my view the actual scaling in the transmitter fulfill the need and people as used to it. Each additional conversion bring inaccuracy's. This may only bring value for some of the toy transmitters wich don't allow scaling. This should likely only used for toy quads.